tests/monster_test_generated.h

```diff
@@ -1025,8 +1025,8 @@ struct Stat FLATBUFFERS_FINAL_CLASS : private flatbuffers::Table {
   bool KeyCompareLessThan(const Stat *o) const {
     return count() < o->count();
   }
-  int KeyCompareWithValue(uint16_t val) const {
-    return static_cast<int>(count() > val) - static_cast<int>(count() < val);
+  int KeyCompareWithValue(uint16_t val_) const {
+    return static_cast<int>(count() > val_) - static_cast<int>(count() < val_);
```

Comment:
That's a generated file, you should update the generator (`idl_gen_cpp.cpp` lines 2001+) and then regenerate all files.